src/server/pkg/worker/master.go

```diff
@@ -261,14 +265,20 @@ nextInput:
 
 // jobManager feeds datums to jobs
 func (a *APIServer) runJob(ctx context.Context, jobInfo *pps.JobInfo, pool *grpcutil.Pool) error {
```

Comment:
I think almost all the logging you added in this file is redundant and not particularly helpful, it's almost all available via things like `inspect-job` and `status`. The one exception might be `egress` which I think we might want a job state for. Other than that though I don't think we should merge the logging in this files, it just makes our log files messier and bigger.